source/docs/variables.md

```diff
@@ -24,11 +24,13 @@ Lodash has been removed from global variables since Hexo 5.0.0. [You-Dont-Need-L
 
 Variable | Description | Type
 --- | --- | ---
-`site.posts` | All posts | `array` of `post` objects
-`site.pages` | All pages | `array` of `page` objects
+`site.posts` | All posts | [Query][queryo] object
+`site.pages` | All pages | [Query][queryo] object
 `site.categories` | All categories | `array` of ???
```

Comment:
`site.categories` and `site.tags` are also Query object. Documentation for other languages can also be updated